packages/app-builder/src/services/feature-access.server.ts

```diff
@@ -142,5 +142,19 @@ export function makeFeatureAccessService({
     }) => {
       return userPermissions.canEditInboxes;
     },
+    getUserRoles: async () => {
+      const licenseEntitlements = await getLicenseEntitlements();
+      if (licenseEntitlements.userRoles) {
+        return ['VIEWER', 'BUILDER', 'PUBLISHER', 'ADMIN'] as const;
+      }
+      return ['ADMIN'] as const;
+    },
+    getInboxUserRoles: async () => {
+      const licenseEntitlements = await getLicenseEntitlements();
+      if (licenseEntitlements.userRoles) {
+        return ['admin', 'member'] as const;
+      }
+      return ['admin'] as const;
+    },
```

Comment:
I wasn't sure about this.

The alternative would have to expose a `isUserRolesAvailable` method like other premium features, and move roles getter in another place.

Reply:
I see,
I don't have a strong opinion on this one, so since you already wrote it like this let's keep it 👍